lib/ranked-model.rb

```diff
@@ -24,14 +24,49 @@ def self.included base
       }
     end
 
+    rails_version = ActiveRecord.version.to_s
+    if rails_version < '5.0'
+      base.module_eval <<-EOS, __FILE__, __LINE__ + 1
+        def handle_ranking_return_value(return_value)
+          @ranked_model_ranking_succeeded
```

Comment:
In Rails < 5 the before_save must return false
In Rails >= 5 the before_save may be aborted by 'throw :abort'
In Rails 5.0 returning 'false' will return a deprecation warning (and will work if the right feature flag is set)

So, because Rails 5.0 is annoying, we make no change to the return value in Rails >= 5.0, and change it to a boolean in < 5.0